### PR TITLE
Initial docs

### PR DESCRIPTION
**Pull request by teruokun**
Putting initial docs in markdown format for the original design and relay/ledger spec.

### CODE REVIEW

File: docs/design/Philosophy.md
Comment (lesiw) [inline docs/design/Philosophy.md:1]: Empty on purpose?

**Comment by lesiw** [comment]
nit: doc names are mixing CamelCase and Kebab-Case (blog-url-case?)

**Comment by lesiw** [comment]
Please squash this into one commit and then I will happily merge it in.

**Comment by teruokun** [comment]
No idea how to squash once it reaches this point, so I've opened a new one with the squashed commit

**Comment by lesiw** [comment]
For future reference, `git rebase -i proof-of-concept`, `squash` all but the latest commit, `pick` that commit, `git push -f pr-branch`

**Pull request closed without merging**